lib/elevator-music.js

```diff
@@ -1,5 +1,6 @@
 var spawn = require('child_process').spawn;
 var musicFile = __dirname + '/../sounds/elevator-music.mp3';
+var bellFile = __dirname + '/../sounds/bell.mp3';
 
 var playIndefinitely = true;
 var player;
```

Comment:
`var bellPlayer` needs to be added here for versions of node that default to strict mode